.storybook/preview.tsx

```diff
@@ -1,14 +1,16 @@
-import { chains, rainbowKitTheme, wagmiClient } from "../src/pages/_app";
 import { RainbowKitProvider } from "@rainbow-me/rainbowkit";
 import { Decorator, Parameters } from "@storybook/react";
 import { initialize, mswDecorator } from "msw-storybook-addon";
 import React from "react";
 import { WagmiConfig } from "wagmi";
 import { GlobalStyle } from "../src/components/GlobalStyle";
+import { chains, rainbowKitTheme, wagmiClient } from "../src/pages/_app";
 import "../src/styles/fonts.css";
 import "./rainbow.css";
 
-initialize();
+initialize({
+  onUnhandledRequest: "bypass",
```

Comment:
this just silences console warnings for api requests that go through without being intercepted by mock service worker.